tests/ci/profile_handler.go

```diff
@@ -97,11 +97,10 @@ func PrepareAccountRoles(token string, accountRolePrefix string, awsRegion strin
 	}
 	args := &EXE.AccountRolesArgs{
 		AccountRolePrefix: accountRolePrefix,
-		// OCMENV            string `json:"ocm_environment,omitempty"`
```

Comment:
I think we need ocm_environment.
IIUC it is used for applying correct trust policies. Right, @nirarg ?

Reply:
Really? I remember it was removed? Let me double check.

Reply:
The `ocm_environment` wasn't removed, there was just added default value `production` in the module, so user doesn't have to indicate this.
We added new data source `rhcs_info` that will replace this requirement, its not planed to be changed in this module, the new WIP module is going to include this

Reply:
Yes,  I checked the code. It is indeed required. Will fix it.

Reply:
Fixed this part. I can create a cluster on production now with env set.
To make all resources using same env, I set it in exec package, rather than profile handler.